submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,105 @@
+
+[Simple handfed mall example, with midgame items?]
+
+A more practical input supply method is using belts, which work very well at any stage of the game. Their supplying is continuous and you do not need input chests when using them. For a good starter mall, all you need to supply is a belt each for iron, copper, and steel; and you can make gears and circuits locally. Belt-based malls have no single design and building them is bit of a puzzle. If you carefully select which machines neighbor each other, you can build a decent compact early-game mall with only 3-4 belts. Alternatively, you can easily design your malls using mini-versions of the main bus technique, as presented [here]([Factorio Tutorials: Factory Shopping Mall - YouTube](https://www.youtube.com/watch?v=MPUcU37hXH4)) by KatherineOfSky and [here]([It is not a MALL; this is a Manufacturing HUB - Factorio 0.18 Tutorial/Guide/How-to - YouTube](https://www.youtube.com/watch?v=PLnv0O3cAnI)) by Nilaus.
+
+[KoS style belt based mall example]
+
+[second example from community?]
+
+The main disadvantage of belt-based malls is that they aren't as compact as other malls, but that does not matter if you have plenty of space. Meanwhile, another possible concern is oddball items with specific ingredients that are unused elsewhere (like small power poles with wood). You can always have a small handfed mall nearby for them.
+
+The third method to supply your mall assembling machines is via logistic robots. This method is unlocked last and relatively expensive but it is very easy to set up and use. For each assembling machine you simply put down a requester (blue) chest for the input and then you can quick-setup the chest's requests via `Shift + Right Click` on the assembling machine followed by `Shift + Left Click` on the chest. You can easily upgrade a handfed mall to be bot-based by just upgrading the chests and adding roboports. 
+
+[bot-based mall example]
+
+[second example from community?]
+
+## **How are the machine outputs controlled and stored?**
+
+The intuitive way to collect the output from your machines is by adding chests. Bigger chests hold more items and passive provider (red) chests will let bots pickup from the mall too. However, you rarely need more than a stack or two of anything at a time. Hence, it would be preferable to prevent overproducing items at your mall and draining your raw materials.
```

Comment:
you introduce here the bot picking up from chests. it would really deserve it being introduced before too, in the general ideas of the mall